.snippets/code/develop/parachains/testing/mock-runtime/pallets-configurations.rs

```diff
@@ -6,4 +7,10 @@ impl frame_system::Config for Test {
     type Hashing = BlakeTwo256;
     type AccountId = u64;
     ...
+}
+
+impl pallet_template::Config for Test {
+	type RuntimeEvent = RuntimeEvent;
+	type WeightInfo = ();
+    ...
```

Comment:
```suggestion
    // ...
```

Reply:
All the reason why it is so important to tackle making the stuff here test-able right after we publish :)